src/components/Global/IframeWrapper/index.tsx

```diff
@@ -1,17 +1,18 @@
-import { useEffect } from 'react'
+import { useEffect, useState } from 'react'
 import Modal from '../Modal'
 
+type IframeWrapperProps = {
+    src: string
+    visible: boolean
+    onClose: () => void
+}
+
 const IframeWrapper = ({
     src,
-    style,
     visible,
     onClose,
-}: {
-    src: string
-    style?: React.CSSProperties
-    visible: boolean
-    onClose: () => void
-}) => {
+}: IframeWrapperProps) => {
+    const [areYouSure, setAreYouSure] = useState(false)
```

Comment:
nitpick (non-blocking): better naming

areYouSure is a little confusing, at first I though it contained the answer of an user, but this is a flag to show confirmation or not, maybe somethin like showConfirmationMessage or something similar would be better